Playlist_Genre_Country.php

```diff
@@ -11,128 +11,128 @@
-					<space></space>
-					<Song_ID1><h4>#</h4></Song_ID1>
-					<images1><h4>Image</h4></images1>
-					<Title1><h4>Title</h4></Title1>
-					<Artist1><h4>Artist</h4></Artist1>
-					<Album1><h4>Album</h4></Album1>
-					<Genre1><h4>Genre</h4></Genre1>
-					<Size1><h4>Size</h4></Size1>
-					<space></space>
-				</heading1>
-			</div>
-
-			<br>
-			<br>
-
-			<!--  <img width="400" height="200" src="<?php include ('/images.php');?>" /> -->
-
-
-
-			<content>
-				<?php
-				require "communityproject_mysqli.php";
-
-				//Creates a variable to store the sql query
-				$query = ("SELECT s.Song_ID, s.Images, s.Title, ar.Artist, al.Album, g.Genre, s.Size
-								FROM songdetails AS s 
-								INNER JOIN Album al ON s.Album_PK = al.Album_PK 
-								JOIN songtoartist j ON s.Song_ID = j.Song_ID 
-								JOIN Artist ar ON ar.Artist_PK = j.Artist_PK 
-								JOIN songtogenre as gs ON s.Song_ID = gs.Song_ID
-								JOIN Genre as g ON g.Genre_PK = gs.Genre_PK
-								WHERE g.Genre = 'Country'
-								OR g.Genre = 'Easy Listening'
-								OR g.Genre = 'Jazz'
-								OR g.Genre = 'Soul'
-								OR g.Genre = 'Mellow'
-								OR g.Genre = 'Vocal'
-								OR g.Genre = 'Blues'
-								OR g.Genre = 'Acoustic'
-								OR g.Genre = 'Bluegrass'
-								ORDER BY s.Song_ID ASC");
-
-
-				//Runs and stores the query using the variable $con (see nav.php) and $query
-				$result = mysqli_query($conn,$query);
-				//runs in a 'while' loop
-				while($output=mysqli_fetch_array($result))
-				{
-				?>
-				<!--php is above. HTML is below. Used to output the query results -->
-
-
-				<heading2>
-					<space></space>
-
-					<Song_ID2><p class = 'white'><?php echo $output['Song_ID']; ?></p></Song_ID2>
-					<Images2><p class = 'white'><?php echo $output['Images']; ?></p></Images2>
-					<Title2><p class = 'white'><?php echo $output['Title']; ?></p></Title2>
-					<Artist2><p class = 'white'><?php echo $output['Artist']; ?></p></Artist2>
-					<Album2><p class = 'white'><?php echo $output['Album']; ?></p></Album2>
-					<Genre2><p class = 'white'><?php echo $output['Genre']; ?></p></Genre2>
-					<Size2><p class = 'white'><?php echo $output['Size']; ?></p></Size2>
-					<space></space>
-
-				</heading2>
-				<?php
+		
+
+		
+		<!-- Container Queries -->
+		<div class="w3-content w3-container w3-padding-64 w3-center w3-yellow w3-image">	
+			<h3 class="w3-center w3-black">COUNTRY</h3>
+
+			<heading1>		
+				<Song_ID1><h4>#</h4></Song_ID1>
+				<image1><h4></h4></image1>
+				<Title1><h4>Title</h4></Title1>
+				<Artist1><h4>Artist</h4></Artist1>
+				<Album1><h4>Album</h4></Album1>
+				<Genre1><h4>Genre</h4></Genre1>
+				<Size1><h4>Size</h4></Size1>
+				<Duration1><h4>Duration</h4></Duration1>
+			</heading1>
+
+			
+			<div class="overflowTest w3-center" >
+				<content>
+					<?php
+						require "communityproject_mysqli.php";
+
+						//Creates a variable to store the sql query
+						$query = ("SELECT s.Song_ID, s.Images, s.Title, 
```

Comment:
new query; created a GROUP_CONCAT to group my genres, artists and albums as well as added seconds within each playlist; like Graeme required